integration-tests/deployment/ccip/view/types/contract_state.go

```diff
@@ -0,0 +1,33 @@
+package types
+
+import (
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type ContractMetaData struct {
+	TypeAndVersion string         `json:"typeAndVersion,omitempty"`
+	Address        common.Address `json:"address,omitempty"`
+	Owner          common.Address `json:"owner,omitempty"`
+}
+
+func NewContractMetaData(tv Meta, addr common.Address) (ContractMetaData, error) {
+	tvStr, err := tv.TypeAndVersion(nil)
```

Comment:
these might not always exist (e.g. WETH) but we can come back to that later, can just be empty in that case